contracts/rbn-staking/IncentivizedVotingLockup.sol

```diff
@@ -92,6 +130,15 @@ contract IncentivisedVotingLockup is
     uint128 blk;
```

Comment:
Just thinking out loud from reading the code. what if instead of creating a new struct `Boost`, we add a new property to the `Point` struct.

```solidity
struct Point {
    int128 bias;
    int128 slope;
    uint128 ts;
    uint128 blk;
    address delegate;
}
```